app/scripts/lib/setupSentry.js

```diff
@@ -64,36 +129,141 @@ export const SENTRY_BACKGROUND_STATE = {
   MetaMetricsController: {
     metaMetricsId: true,
     participateInMetaMetrics: true,
+    eventsBeforeMetricsOptIn: true,
   },
   NetworkController: {
-    networkId: true,
     networkStatus: true,
+    selectedNetworkClientId: false,
+    networkId: true,
     providerConfig: {
-      nickname: true,
-      ticker: true,
+      rpcUrl: false,
       type: true,
+      chainId: false,
+      ticker: true,
+      nickname: true,
+      rpcPrefs: false,
+      id: false,
+    },
+    networkConfigurations: false,
+    networksMetadata: {
+      EIPS: true,
+      status: true,
     },
   },
+  NftController: {
+    allNftContracts: false,
+    allNfts: false,
+    ignoredNfts: false,
+  },
   OnboardingController: {
     completedOnboarding: true,
     firstTimeFlowType: true,
     seedPhraseBackedUp: true,
   },
   PreferencesController: {
-    currentLocale: true,
     featureFlags: true,
-    forgottenPassword: true,
+    // double-check if contains PID?
     ipfsGateway: true,
-    preferences: true,
+    identities: false,
+    lostIdentities: false,
+    selectedAddress: false,
+    useTokenDetection: true,
+    useNftDetection: true,
+    openSeaEnabled: true,
+    isMultiAccountBalancesEnabled: true,
+    disabledRpcMethodPreferences: true,
+    showTestNetworks: true,
+    isIpfsGatewayEnabled: true,
     useBlockie: true,
     useNonceField: true,
     usePhishDetect: true,
+    dismissSeedBackUpReminder: true,
+    useMultiAccountBalanceChecker: true,
+    use4ByteResolution: true,
+    useCurrencyRateCheck: true,
+    advancedGasFee: true,
+    incomingTransactionsPreferences: true,
+    // double-check if contains PID?
+    knownMethodData: true,
+    currentLocale: true,
+    forgottenPassword: true,
+    preferences: {
+      autoLockTimeLimit: true,
+      showFiatInTestnets: true,
+      showTestNetworks: true,
+      useNativeCurrencyAsPrimaryCurrency: true,
+      hideZeroBalanceTokens: true,
+    },
+    useAddressBarEnsResolution: true,
+    infuraBlocked: true,
+    ledgerTransportType: true,
+    snapRegistryList: true,
+    transactionSecurityCheckEnabled: true,
+    theme: true,
+    isLineaMainnetReleased: true,
+
+    securityAlertsEnabled: false,
+    snapsAddSnapAccountModalDismissed: true,
   },
   SignatureController: {
+    unapprovedMsgs: false,
```

Comment:
These would reveal pending signatures, which isn't something we otherwise expose. These can be omitted.